### PR TITLE
CO-3086 mass_mailing_switzerland migration v11

### PR DESCRIPTION
**Pull request by quentingigon**

### CODE REVIEW

**Comment by quentingigon** [comment]
TODO: change oca_dependencies to add mail_sendgrid_mass_mailing to social

**Comment by quentingigon** [comment]
waiting for the fix for sms_939 from JosueBulle

File: oca_dependencies.txt [hunk 1]
```diff
@@ -27,7 +27,7 @@ geospatial https://github.com/josueBulle/geospatial mix
 queue
 report-print-send
 server-ux
-social https://github.com/CompassionCH/social 11.0-mig-mail_sendgrid
+social https://github.com/quentingigon/social sendgrid_and_mass_mailing_migrations
```
Comment (ecino) [inline oca_dependencies.txt]: You can change this as we created a branch on CompassionCH
```suggestion
social https://github.com/CompassionCH/social 11.0-mig-mail_sendgrid_mass_mailing
```

File: mass_mailing_switzerland/models/mass_mailing.py [hunk 2]
```diff
@@ -29,16 +28,16 @@ class MassMailing(models.Model):
     partner_test_sendgrid_id = fields.Many2one('res.partner',
                                                'Test Partner')
 
-    _sql_constraints = [('slug_uniq', 'unique (mailing_slug)',
-                         'You have to choose a new slug for each mailing !')]
+    # _sql_constraints = [('slug_uniq', 'unique (slug)',
+    #                      'You have to choose a new slug for each mailing !')]
```
Comment (ecino) [inline mass_mailing_switzerland/models/mass_mailing.py]: Why removing this?

File: mass_mailing_switzerland/migrations/10.0.2.1.2/pre-migration.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```
Comment (ecino) [inline mass_mailing_switzerland/migrations/10.0.2.1.2/pre-migration.py:1]: You can remove the migrations folder

**Review by ecino — CHANGES_REQUESTED** [review]

File: mass_mailing_switzerland/models/mass_mailing.py [hunk 4]
```diff
@@ -29,7 +29,7 @@ class MassMailing(models.Model):
     partner_test_sendgrid_id = fields.Many2one('res.partner',
                                                'Test Partner')
 
-    _sql_constraints = [('slug_uniq', 'unique (mailing_slug)',
+    _sql_constraints = [('slug_uniq', 'unique (slug)',
                          'You have to choose a new slug for each mailing !')]
```
Comment (ecino) [inline mass_mailing_switzerland/models/mass_mailing.py]: In fact I checked and this field was removed! So you can remove the constraint as you did before, sorry.

**Review by ecino — CHANGES_REQUESTED** [review]

**Comment by ecino** [comment]
flake8 error :
```
mass_mailing_switzerland/models/mass_mailing.py:14:1: E303 too many blank lines (3)
```

**Comment by ecino** [comment]
Please rebase the branch to fix the conflicts.

**Review by ecino — APPROVED** [review]

**Pull request merged**